orchestra/todos/views.py

```diff
@@ -33,12 +33,12 @@
               logger=logger)
 def update_todos_from_todolist_template(request):
     todolist_template_slug = request.data.get('todolist_template')
-    task_id = request.data.get('task')
+    project_id = request.data.get('project')
+    step_slug = request.data.get('step')
```

Comment:
is is step-slug, or does `_set_step_to_validated_data` make it the actual step?

Reply:
Good catch! Thank you! I added `step__slug=step_slug` bit to the filter.

As a side note, validation is not available, when `data=` is not passed: https://github.com/encode/django-rest-framework/blob/e39d8410de26003421f7e644f9206833a3b9bbe0/rest_framework/serializers.py#L73